### DISCUSSION TITLE
SassError: Invalid CSS after ".next-+": expected selector, was '"formily-item"'

### DISCUSSION THREAD

**Discussion by c78702505** [alibaba/formily#1438] [Q&A 2.x] [answered]
ERROR  Failed to compile with 1 errors                                                                          上午9:49:10

 error  in ./node_modules/@formily/next/esm/form-item/main.scss

Module build failed (from ./node_modules/sass-loader/dist/cjs.js):
SassError: Invalid CSS after ".next-+": expected selector, was '"formily-item"'
        on line 1 of node_modules/@formily/next/esm/form-item/grid.scss
        from line 3 of /Users/river/projects/work/cmps-react/node_modules/@formily/next/esm/form-item/main.scss
>> .next-+"formily-item"
   ------^
   
     @@调整去掉`“”`后正常
./node_modules/@formily/next/esm/form-item/scss/variable.scss
>> .next-+formily-item

编译器：
"node-sass": "^4.14.1",
"sass-loader": "^8.0.2"

**Comment by janryWang** [comment, marked as answer]
使用sass，不要用node-sass
